packages/vhd-lib/src/_resolveAlias.js

```diff
@@ -1,7 +1,7 @@
 import resolveRelativeFromFile from './_resolveRelativeFromFile'
 
 export function isVhdAlias(filename) {
-  return filename.endsWith('.alias.vhd')
+  return filename?.endsWith('.alias.vhd') ?? false
```

Comment:
How can this happen?

Reply:
after more testing, it only happenned because I use resolve wrongly. I will revert this change